[2025-06-04T09:20:21.552+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-04T09:20:21.578+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_postgres_create_table_and_read_dag.create_postgres_table manual__2025-06-04T09:20:20.612999+00:00 [queued]>
[2025-06-04T09:20:21.590+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_postgres_create_table_and_read_dag.create_postgres_table manual__2025-06-04T09:20:20.612999+00:00 [queued]>
[2025-06-04T09:20:21.590+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-04T09:20:21.606+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_postgres_table> on 2025-06-04 09:20:20.612999+00:00
[2025-06-04T09:20:21.610+0000] {standard_task_runner.py:63} INFO - Started process 34 to run task
[2025-06-04T09:20:21.613+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_postgres_create_table_and_read_dag', 'create_postgres_table', 'manual__2025-06-04T09:20:20.612999+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/test_postgres_conn_and_create_table.py', '--cfg-path', '/tmp/tmpy5anwbyn']
[2025-06-04T09:20:21.614+0000] {standard_task_runner.py:91} INFO - Job 138: Subtask create_postgres_table
[2025-06-04T09:20:21.659+0000] {task_command.py:426} INFO - Running <TaskInstance: test_postgres_create_table_and_read_dag.create_postgres_table manual__2025-06-04T09:20:20.612999+00:00 [running]> on host 8b60c3c987d2
[2025-06-04T09:20:21.752+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_postgres_create_table_and_read_dag' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-04T09:20:20.612999+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-04T09:20:20.612999+00:00'
[2025-06-04T09:20:21.754+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-04T09:20:21.781+0000] {base.py:84} INFO - Using connection ID 'datalake_***' for task execution.
[2025-06-04T09:20:21.868+0000] {logging_mixin.py:188} INFO - ✅ Tạo schema và bảng thành công hoặc đã tồn tại.
[2025-06-04T09:20:21.874+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-04T09:20:21.876+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-04T09:20:21.911+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_postgres_create_table_and_read_dag, task_id=create_postgres_table, run_id=manual__2025-06-04T09:20:20.612999+00:00, execution_date=20250604T092020, start_date=20250604T092021, end_date=20250604T092021
[2025-06-04T09:20:22.428+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-04T09:20:22.448+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-04T09:20:22.451+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
